pkg/log/log_file.go

```diff
@@ -164,6 +166,8 @@ func checkLogFiles() {
 			warningLogFiles = append(warningLogFiles, file)
 		} else if strings.Contains(file.Name(), ".log.ERROR.") {
 			errorLogFiles = append(errorLogFiles, file)
+		} else if strings.Contains(file.Name(), ".log.FATAL") {
+			fatalLogFIles = append(fatalLogFIles, file)
```

Comment:
Thanks for the fix on rotating FATAL logs! I want to confirm if we don't need the dot after FATAL.

Reply:
`log.FATAL` is enough. But for uniformity, i will add the missing `.`. Thanks